crates/core/src/operations/subscribe.rs

```diff
@@ -279,16 +279,33 @@ impl Operation for SubscribeOp {
                         );
                     }
 
+                    // SUBSCRIPTION_DIAG: Log subscription attempt
```

Comment:
let's lower the debug level of all this debugging logs

Reply:
The debug logs in subscribe.rs were added for diagnosing this specific issue and can be lowered or removed in a follow-up PR. They helped identify the root cause of the gateway storage problem.